src/modules/status.js

```diff
@@ -0,0 +1,22 @@
+import LocalStorage from './local_storage.js';
+
+const localS = new LocalStorage();
+
+class Status {
+    updateTaskCompletedStatus = (index, status) => {
+      const todo = localS.getLocalStorage();
+      todo[index].completed = status;
+      localStorage.setItem('todo', JSON.stringify(todo));
+    }
+
+    clearTaskCompleted = () => {
+      const todo = localS.getLocalStorage();
+      // for (let i = 0; i < todo.length; i += 1) {
+      //     todo[i].index = i;
+      // }
```

Comment:
- [ ] Please remove this commented code because it is a bad practice comments should be reserved to explain difficult-to-understand code or the reason for doing something out of the ordinary. Don't worry code should be stored in source control :+1: